workflow-templates/update-browserslist.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Auto Update Browserslist Workflow",
```

Comment:
nit: "Auto Update Browserslist **DB** Workflow" maybe?

Reply:
Similarly, should the file name also include DB, e.g. `update-browserslist-db.properties.json`?